url.bs

```diff
@@ -164,11 +164,11 @@ contains bytes that are not <a>ASCII bytes</a> might be insecure and is not reco
 
      note that query and application/x-www-form-urlencoded use their own
      local sets -->
-<p>The <dfn>simple encode set</dfn> are <a>C0 controls</a> and all code points greater
-than U+007E.
+<p>The <dfn>C0 control encode set</dfn> are <a>C0 controls</a> and all code points greater than
```

Comment:
No, I didn't remember both were used for percent encoding. Is there a succinct description of where this is used, like "path" or "userinfo"? "C0 control", is what it *is*, not how it's used.